.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1 +1,46 @@
-Closes #[ISSUE-NUM].
+# Info
+
+Closes **[ISSUE NUMBER]**. (If there is no issue for this pull request yet, please create one or
+delete this line if the pull request is for a very minor tweak).
+
+# Description
+
+What changes did you make? List all distinct problems that this PR addresses. Explain any relevant
+motivation or context.
+
+[description]
+
+## Changes
+
+- [Fill in here]
+
+# Type of Change
+
+- [ ] Bug Fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
+      expected)
+- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
+      linting/formatting)
+- [ ] Continuous Integration/DevOps Change (Related to deployment steps or continuous integration
+      workflows)
+- [ ] Other: (Fill In) <!-- Edit this type of change if you select this -->
```

Comment:
Here I think it's good to mark which one you should update the package version for (major minor patch) so it's straightforward for someone to see if/how they should update the version

e.g.
```md
- [ ] Patch (non-breaking change/bugfix)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Documentation (A change to a README/description)
- [ ] Continuous Integration/DevOps Change (Related to deployment steps, continuous integration
      workflows, linting, etc.)
- [ ] Other: (Fill In) <!-- Edit this type of change if you select this -->

If you've selected Patch, Minor, or Major as your change type, make sure to bump the version before merging in package.json!
```